magnum_cluster_api/resources.py

```diff
@@ -1440,6 +1440,8 @@ def labels(self) -> dict:
         return {**super().labels, **labels}
 
     def get_or_none(self) -> objects.Cluster:
+        if self.cluster.stack_id is None:
+            return None
```

Comment:
i think this is not necessary because we're doing `get_or_none` so if it cant find it, it will just return `None` anyways.